colin-api/src/colin_api/models/filing.py

```diff
@@ -349,6 +350,23 @@ def _insert_filing(cls, cursor, filing, ar_date: str, agm_date: str):
                     filing_type_code=filing_type_code,
                     effective_dt=filing.effective_date
                 )
+            elif filing_type_code in ['NOALA', 'NOALB', 'NOALC', 'NOALE', 'NOALR', 'NOALU']:
+                arragement_ind = 'N'
+                court_order_num = None
+                if court_order := filing.body.get('courtOrder', None):
+                    arragement_ind = 'Y' if court_order.get('effectOfOrder', None) else 'N'
+                    court_order_num = court_order['fileNumber']
+
+                insert_stmnt = insert_stmnt + ', arrangement_ind, court_order_num, ods_typ_cd) '
+                values_stmnt = values_stmnt + ", :arragement_ind, :court_order_num, 'F')"
+                cursor.execute(
+                    insert_stmnt + values_stmnt,
```

Comment:
i can see this follows the previous pattern, to note that bind variables are a better preference.
this should open a techdebt tix to convert all of these.

Reply:
ok

Reply:
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/6947